sql/moz-fx-data-shared-prod/stripe_external/customers_changelog_v1/query.sql

```diff
@@ -41,7 +41,7 @@ customer_latest_discounts AS (
           coupons.currency,
           coupons.duration,
           coupons.duration_in_months,
-          PARSE_JSON(coupons.metadata) AS metadata,
+          metadata,
```

Comment:
seems like this should be
```suggestion
          coupons.metadata,
```